### PR TITLE
Support direct URL navigation

### PR DESCRIPTION
**Pull request by MattRCole**
Resolves issues in #4! Direct URL navigation should now be supported!

### CODE REVIEW

**Review by pserwylo — APPROVED** [review]
Looks good, lets give it a whirl and see how it goes.

File: src/App.tsx [hunk 1]
```diff
@@ -11,8 +11,8 @@ function App() {
 
   return (
       <div className="App">
-        <BrowserRouter>
-          <Route path="/">
+        <BrowserRouter basename="/web-lexica">
```
[the remark this answers is not in the document]
Comment (MattRCole) [reply src/App.tsx:14]: It looks like it's set up to handle that correctly actually. I'm surprised that it does.

**Comment by pserwylo** [comment]
Screenshot_20210913_112139_org mozilla fennec_fdroid

**Comment by MattRCole** [comment]
Yes! :D

**Pull request merged**